versions.tf

```diff
@@ -1,10 +1,17 @@
 terraform {
-  required_version = ">= 0.12.0, < 0.14.0"
-
   required_providers {
-    aws      = ">= 2.0, < 4.0"
-    template = "~> 2.0"
-    local    = "~> 1.2"
-    null     = "~> 2.0"
+    aws = {
```

Comment:
The new `required_providers` syntax was introduced for backwards compatibility in v0.12.26 -- Can you update the lower bound `required_version` to reflect that? Or don't update these at all and instead just remove the upper bound `required_version`.

Reply:
pinned lower version to `v0.12.26`